plugins/coral-plugin-facebook-auth/README.md

```diff
@@ -0,0 +1,26 @@
+# coral-plugin-facebook-auth
+
+This plugin provides facebook authentication support for Talk.
+
+## Configuration
+
+This plugin looks for the following configuration from the environment:
+
+- `TALK_FACEBOOK_APP_ID` (*required*) - the Facebook app id for your Facebook
+Login enabled app.
+- `TALK_FACEBOOK_APP_SECRET` (*required*) - the Facebook app secret for your
+Facebook Login enabled app.
+
+### License
+
+    Copyright 2016 Mozilla Foundation
```

Comment:
is our stuff copyrighted under mozilla or the Coral Project as a separate identity? @losowsky